packages/react/src/primitives/StepperField/StepperField.tsx

```diff
@@ -35,6 +34,8 @@ const StepperFieldPrimitive: Primitive<StepperFieldProps, 'input'> = (
     isDisabled,
     isReadOnly,
     isRequired,
+    increaseButtonLabel = SharedText.StepperField.ariaLabel.IncreaseTo,
+    decreaseButtonLabel = SharedText.StepperField.ariaLabel.DecreaseTo,
```

Comment:
Same here, I prefer `increaseButtonLabel`

Reply:
I can make the changes for pagination label. But I think for the other ones, it would be a breaking change I think? Actually that's the reason why I didn't refactor the `ShareText` structure to make it more consistent because I found that we export it in our bundle, which we shouldn't IMO 🤣